tools/sdk-readme-table-generator/TableGenerator/Program.fs

```diff
@@ -8,19 +8,19 @@ open TableGenerator.Table
 let inputBranches =
         [ { GitBranchName = "master"
             DisplayName = "Master<br>(6.0.x&nbsp;Runtime)"
-            AkaMsChannel = Some("net6/dev") }
+            AkaMsChannel = Some("6.0/daily") }
           { GitBranchName = "release/6.0.1xx-preview1"
             DisplayName = "Release/6.0.1xx-preview1<br>(6.0.x&nbsp;Runtime)"
-            AkaMsChannel = Some("net6/preview1") }
+            AkaMsChannel = Some("6.0-preview1/daily") }
           { GitBranchName = "release/5.0.3xx"
             DisplayName = "Release/5.0.3XX<br>(5.0 Runtime)"
-            AkaMsChannel = Some("net5/5.0.3xx/daily") }
+            AkaMsChannel = Some("5.0.3xx/daily") }
```

Comment:
Same issue with this pattern: https://aka.ms/dotnet/5.0.3xx/daily/Sdk/dotnet-sdk-win-x64.exe

Reply:
This channel was just created. Nothing is currently getting an aka.ms link from 503 yet